packages/taraxa-ui/src/components/Header/Header.stories.tsx

```diff
@@ -1,12 +1,39 @@
 import React from 'react';
 import { Story, Meta } from '@storybook/react';
 import Header, { HeaderProps } from './Header';
+import { Taraxa } from '../Icons';
+import Button from '../Button';
+import NetworkMenu from '../NetworkMenu/NetworkMenu';
 
 export default {
-  title: 'Components/Header',
+  title: 'v1/Header',
```

Comment:
Do we need to keep this v1 prefix?

Reply:
No, we don't this would just group the story elements under the `v1` category, but we don't need that so it's not necessary